doc/build.md

```diff
@@ -35,8 +35,10 @@ These options control some general build configuration values.
 | multilib                    | true    | Build every multilib configuration supported by the compiler                         |
 | multilib-list               | <empty> | If non-empty, the set of multilib configurations to compile for                      |
 | multilib-exclude            | <empty> | Multilib configurations containing any of these strings will not be built            |
+| sanitize-undefined          | false   | Build the library with -fsanitize=undefined                                          |
```

Comment:
There is a built-in meson option b_sanitize that might also work here?

Reply:
ooo. I had no idea, I'll go take a look. Thanks!

Reply:
Yup, that looks a bunch cleaner.